test/common/session/BUILD

```diff
@@ -1,3 +1,6 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+load("//bazel:bazel.bzl", "authsvc_cc_test")
+
 cc_library(
```

Comment:
does this have to be `cc_library`? Can this be `authsvc_cc_library` as the rest?